src/ainftBase.ts

```diff
@@ -13,8 +13,9 @@ export default class AinftBase {
     route?: string,
   ) {
     this.route = route || '';
-    this.baseUrl = baseUrl + this.route;
     this.ain = ain;
+
+    this.setBaseUrl(baseUrl);
```

Comment:
ah, what about making baseUrl optional as it already has a default value?

Reply:
Done!

Reply:
👍